app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -22,6 +22,7 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
             $scope.items = data.data.items;
             $scope.relations = data.data.relations;
             $scope.kinds = data.data.kinds;
+            $scope.icons = data.data.icons;
```

Comment:
For being able to test it properly, iirc, the `getIcon` didn't work properly in this context w/o it: https://github.com/ManageIQ/manageiq/pull/8630/files#diff-73ef271f9f62c257d152ab3fa2b11221R109